### ISSUE TITLE
Docs site needs an FAQ section

### ISSUE THREAD

**Issue by yurishkuro** [opentracing/opentracing.github.io#134]
I was just looking through the open issues in the repo, and many of them are the questions that come again and again either in issues or on gitter, such as "why sampling is not in the spec", etc. Most of those issues have been answered and should really be closed as "wontfix", but closed issues are not a very friendly way for people to learn the answers to these common questions. I think we should start moving them to an FAQ page.

@bensigelman @pritianka

**Comment by pritianka** [comment]
+1 to FAQ page. I had it in my original spec but we didn't have time to
implement yet. Should I create a google doc for us to put in the content
until there's enough to release an FAQ page?

Priyanka Sharma
Advisor, HeavyBit <http://heavybit.com/>
Head of Marketing & Partnerships, LightStep <http://lightstep.com/>
Co-founder, WakaTime <http://www.wakatime.com/> - Quantify your coding
www.twitter.com/pritianka
650-796-7125

**Comment by yurishkuro** [comment]
Yes, google doc sounds fine. We can even link to it from the site for the time being.

**Comment by wu-sheng** [comment]
FAQ page will be very helpful. Now I am translating OpenTracing doc to Chinese, many Chinese users are interested, may have so many questions to ask me. 

Please keep me notice, if there are any further progress.

**Comment by yurishkuro** [comment]
@wu-sheng do you want to start adding your question on this ticket? We can decide if they qualify for the FAQ page.

**Comment by wu-sheng** [comment]
I am very glad to do. Several issues I created are from them.
such as:
1. #155 in-band and out-of band are not easy to understand for them.
2. what should application developer to do, if they want to follow OpenTracing specification.
3. issue https://github.com/opentracing/opentracing-java/issues/63, I post on grpc-java

And OpenTracing is very new idea for application developers, and framework developers. They need time to take and follow it. 

Many of them hope OpenTracing can do something more than a specification or guide, and it can provide a solution to integrate tracer system, APM, framework library and application eco system. All of these part can provide **spans**,  and diff tracer and APM system can collect them , store them, and analysis them, or more.

I am very interesting in development direction of OpenTracing, and want to help OpenTracing to gain influence in China.  And if there is the FAQ page, I will continue to translate the page to Chinese, it will be very helpful for Chinese developers.

Is there any opportunities, I can join OpenTracing org or Discussion Group ( or in other ways )? Help me to know the direction earlier, or more fully understand the core goal?

**Comment by Shreya1771** [comment]
Hi, @pritianka @yurishkuro I would like to work on this issue. :-)
I would Like to know more about what FAQ would contain like.
Thanks!
